stable/ambassador/values.yaml

```diff
@@ -5,8 +5,8 @@
 replicaCount: 3
 daemonSet: false
 
-ambassador:
-  id: default
+# ambassador:
```

Comment:
Maybe comment above this saying "# You only need to change this if you're running multiple Ambassadors in one cluster" ?